students/minji/users/views.py

```diff
@@ -1,3 +1,39 @@
 from django.shortcuts import render
+import json
+import re
 
-# Create your views here.
+from django.http import JsonResponse
+from django.views import View
+
+from users.models import User
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data                = json.loads(request.body)
+            email               = data["email"]
+            password            = data["password"]
+            email_validation    = re.compile("^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$")
+            password_validation = re.compile("^.*(?=^.{8,}$)(?=.*\d)(?=.*[a-zA-Z])(?=.*[!@#$%*^&+=].*$)")
```

Comment:
```suggestion
            REGEX_EMAIL    = re.compile("^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$")
            REGEX_PASSWORD = re.compile("^.*(?=^.{8,}$)(?=.*\d)(?=.*[a-zA-Z])(?=.*[!@#$%*^&+=].*$)")
```

검증에 대한 정규표현식은 주로 변하지 않는 값이므로 상수로 선언됩니다.
python coding convention에서 상수는 대문자로, 필요하면 `_` 을 사용하도록 권고 하고있습니다.

추가로 정규표현식을 사용할 경우, 자주 사용되는 경우가 아니라면 `compile()`해서 사용하지 않고, `match()`를 사용하는 것을 공식 문서에서 추천하고 있습니다.
```python
import re

. . .

REGEX_EMAIL    = ‘^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$’
REGEX_PASSWORD = ‘^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$’

. . .

if not re.match(REGEX_EMAIL, value):
    return JsonResponse(...)
if not re.match(REGEX_PASSWORD, value):
    return JsonResponse(...)
```

위 두가지 반영해서 수정해주세요~!